### PR TITLE
Student Model added

### PR DESCRIPTION
**Pull request by kahyalar**

### CODE REVIEW

File: Attend/Models/Student.swift [hunk 1]
```diff
@@ -0,0 +1,25 @@
...
+//
+
+import Foundation
+
+class Student
+{
+    let fullName: String
+    let studentNumber: String
+    let department: String
+    let classNo: Int
```
Comment (desturrr) [inline Attend/Models/Student.swift]: Do we really put classNo inside Student model ?

File: Attend/Models/Student.swift [hunk 2]
```diff
@@ -0,0 +1,25 @@
...
+//
+//  Created by Furkan Kahyalar on 10/11/2017.
+//  Copyright © 2017 suat.karakusoglu. All rights reserved.
+//
+
+import Foundation
+
+class Student
+{
+    let fullName: String
```
Comment (desturrr) [inline Attend/Models/Student.swift]: firstName, lastName should be separated.
There can be calculated property to give you full name.

**Comment by kahyalar** [comment]
Student model refactored:
* Created 2 new let values: firstName, lastName
* fullName attribute change it let to var
* fullName is declared by firstName + " " + fullName inside of init func

File: Attend/Models/Student.swift (hunk 1 above)
Comment (kahyalar) [reply Attend/Models/Student.swift]: I actually looked this information from our transcript. It can be removed, if you want.

**Pull request merged**